docs/quick-start.MD

```diff
@@ -39,6 +39,69 @@ kubectl get pods
 
 ```
 
+
+## Installing by Kustomize
+
+You can install operator using [Kustomize](https://kubernetes-sigs.github.io/kustomize/) by pointing to the remote kustomization file.
+
+```yaml
+# kustomization.yml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- namespace.yml
+- github.com/VictoriaMetrics/operator/config/default?ref=v0.2.0
+
+namespace: monitoring
+images:
+- name: victoriametrics/operator
+  newTag: v0.2.0
+
+patches:
+- manager.patch.yml
+```
+
+```yaml
+# namespace.yml
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: monitoring
+```
+
+Change configuration for operator at `manager.patch.yml`, possible settings: [operator-settings](/vars.MD)
+
+```yaml
+# manager.patch.yml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: vm-operator
+  name: vm-operator
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        env:
+        - name: WATCH_NAMESPACE
+          value: "app-prod"
```

Comment:
There is a problem with WATCH_NAMESPACE, currently operator was tested only with WATCH_NAMESPACE="" aka all namespaces.

 So, i think, its better to introduce some other parameter, for instance:
```- name: VM_CLUSTERDOMAINNAME
      value: cluster.local
```

Reply:
sure no problem